cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -112,6 +116,24 @@ def test_snapshot_crud(small_test_vm, provider):
     snapshot.delete()
 
 
+@pytest.mark.uncollectif(lambda provider: not provider.one_of(RHEVMProvider))
+@pytest.mark.meta(automates=[BZ(1384517)])
+def test_create_without_description(small_test_vm):
+    """
+    Test that we get an error message when we try to create a snapshot with
+    blank description on RHV provider.
+
+    Metadata:
+        test_flag: snapshot, provision
+    """
+    snapshot = new_snapshot(small_test_vm, has_name=False, description=False)
+    with pytest.raises(AssertionError):
+        snapshot.create()
+    view = snapshot.vm.create_view(InfraVmSnapshotAddView)
+    assert view.is_displayed
+    view.flash.assert_message('Description is required')
```

Comment:
@quarckster I solved it like this, hope this is more in line with the general design philosophy of the framework

Reply:
@jan-zmeskal  yes, this is correct